benches/benchmark_main.rs

```diff
@@ -2,7 +2,7 @@ use criterion::criterion_main;
 
 mod benchmarks;
 criterion_main! {
-    benchmarks::ipa_verify_multi_exp::benches,
```

Comment:
This file was removed since it's now a subbenchmark of `ipa_prove`. 
They were identical in all the setup but just calling a different flavor of `check_**(..)`.

I'll comment again about this later.